helpers/ifrauBackdropService.js

```diff
@@ -0,0 +1,9 @@
+export async function getIfrauBackdropService(ifrauBackdropService) {
+	if (!window.ifrauclient) return;
+	if (ifrauBackdropService) return ifrauBackdropService;
+
+	const ifrauClient = await window.ifrauclient().connect();
+	ifrauBackdropService = await ifrauClient.getService('dialogWC', '0.1');
```

Comment:
I think you'll want to define `ifrauBackdropService` in this file instead of passing it in?